### PR TITLE
created markdown file for cpp vector front

### LINKED ISSUE #4983: [Term Entry] C++ vector .front()
### Reason/inspiration (optional)

"""We would like a new entry on the `.front()` term entry in Cpp under `Vector` concept. The entry should go in a new file under `content/cpp/concepts/vectors/terms/front/front.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept.
- An `Example` section that provides an example demonstrating the concept in use.

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."""

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by evanshi0811**
### Description

Created a markdown file for the front method for C++ vectors.

### Issue Solved

Closes #4983

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/cpp/concepts/vectors/terms/front/front.md
Comment (cigar-galaxy82) [inline content/cpp/concepts/vectors/terms/front/front.md:44]:
```suggestion
#include <iostream>
#include <vector>

int main() {
  // Declaring a vector of integers
  std::vector<int> numbers = {1, 2, 3, 4};

  // Print out first element
  std::cout << numbers.front();

  return 0;
}
```

File: content/cpp/concepts/vectors/terms/front/front.md
Comment (cigar-galaxy82) [inline content/cpp/concepts/vectors/terms/front/front.md:53]: Put this after the Codebyte Example heading

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]

**Comment by cigar-galaxy82** [comment]
Hi @evanshi0811 thank you for contributing to Codecademy docs, I have suggested a few changes please modify them.

File: content/cpp/concepts/vectors/terms/front/front.md
Comment (evanshi0811) [reply content/cpp/concepts/vectors/terms/front/front.md]: Resolved in latest commit.

**Comment by evanshi0811** [comment]
Is there anything else I need to change before moving onto the second review?

**Review by mamtawardhani — COMMENTED** [review]
Thank you for contributing to Codecademy Docs @evanshi0811 😄 

The entry is ready to be merged! 🚀

**Pull request merged**
